packages/ui/src/components/library/Autocomplete.tsx

```diff
@@ -0,0 +1,141 @@
+import { Autocomplete as AutocompleteMui } from '@mui/material'
+import React, { SyntheticEvent } from 'react'
+import { styled } from '@mui/material/styles'
+import { AutocompleteRenderInputParams } from '@mui/material/Autocomplete/Autocomplete'
+import { HiOutlineChevronDown } from 'react-icons/hi2'
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types'
+
+interface AutocompleteProps {
+  InputProps?: Partial<AutocompleteRenderInputParams['InputProps']>
+  className?: string
+  isOptionEqualToValue?: (option: any, value: any) => boolean
+  filterOptions?: (options: any[], state: any) => any[]
+  options: any[]
+  getOptionLabel: (option: any) => string
+  onChange?: (event: any, value: any) => void
+  value?: any
+  renderOption: (props: any, option: any, state: any) => React.ReactNode
+  disableClearable?: boolean
+  onKeyDown?: (e: any) => void
```

Comment:
there are many `any`s here, can we import the MUIs types?

Reply:
The `renderOption`, `onChange`, `isOptionEqualToValue` etc above still have some `any`.